SPECS/msft-golang/msft-golang.spec

```diff
@@ -14,7 +14,7 @@
 %define __find_requires %{nil}
 Summary:        Go
 Name:           msft-golang
-Version:        1.24.5
+Version:        1.24.7
```

Comment:
I think we should also release-bump all specs with `BuildRequires` on `msft-golang`, so that they get rebuilt with the new version.